src/layers/legacy/medCoreLegacy/views/medAbstractLayeredView.cpp

```diff
@@ -238,7 +238,6 @@ void medAbstractLayeredView::removeData(medAbstractData *data)
 
     if( res > 0 )
     {
-        emit layerRemoved(layer);
```

Comment:
connected to a method witch try working on a delete data, creating the previous "Unable to retrieve data at layer: 0 from: "medVtkView"" warning. Line not needed